omod/src/main/java/org/openmrs/module/reportingrest/web/resource/ReportRequestResource.java

```diff
@@ -194,4 +196,18 @@ public void setProperty(Object instance, String propertyName, Object value) thro
 	private ReportService getService() {
 		return Context.getService(ReportService.class);
 	}
+	
+	@Override
+	public SimpleObject asRepresentation(ReportRequest delegate, Representation representation) throws ConversionException {
+		DelegatingResourceHandler<ReportRequest> handler = (DelegatingResourceHandler<ReportRequest>) getResourceHandler(
+		    delegate);

```

Comment:
@gitcliff instead of this, shouldn't you just add a ref representation like for the rest of the webservices rest resources?